tests/likelihood/gauss_family/test_const_gaussian.py

```diff
@@ -99,3 +254,123 @@ def test_after_read_all_statistics_are_ready(
     for gs in likelihood.statistics:
         stat: Statistic = gs.statistic
         assert stat.ready
+
+
+def test_make_realization_chisq(
+    trivial_stats,
+    sacc_data_for_trivial_stat: sacc.Sacc,
+    tools_with_vanilla_cosmology: ModelingTools,
+):
+    likelihood = ConstGaussian(statistics=trivial_stats)
+    likelihood.read(sacc_data_for_trivial_stat)
+    params = firecrown.parameters.ParamsMap(mean=10.5)
+    likelihood.update(params)
+    likelihood.compute_chisq(tools_with_vanilla_cosmology)
+
+    new_sacc = likelihood.make_realization(sacc_data_for_trivial_stat)
```

Comment:
Consider adding pytest-rerunfailures to our list of required Conda packages, and decorating this test with `@pytest.mark.flaky(reruns=2)`. This should reduce the random failure probability from 1 in a million to a more comfortable 1 in a trillion.

This decoration should probably be done to all the tests that are statistical in nature.